manifests/GitLab/Gitter.IM/5.1.0.yaml

```diff
@@ -0,0 +1,15 @@
+Id: Gitlab.Gitter.IM
+Version: 5.1.0
+Name: Gitter IM
+Publisher: Gitlab
+License: 2011-2017 GitLab B.V.
+LicenseUrl: https://gitlab.com/gitlab-org/gitter/desktop/
+AppMoniker: gitter
+Description: Great IM Client for Github
+Homepage: https://gitter.im
+Tags: "Chat, messages"
+Installers:
+  - Arch: x86
+    Url: https://update.gitter.im/win/latest
+    Sha256: f80af1656b2b365069ffc5f839db7b9995b6b6782ea6fbb810c65aed839e2814
```

Comment:
`Sha256: 167BFBDA8FF109BCE354D6A5777718431A32298CB0189472F4CEC31DAC7A068D`

for the Gitter-5.0.1.

Reply:
Thank you. 😄

Reply:
Thank you for contribution. 🥂  I'm add another review btw.